osgar/tools/lidarview.py

```diff
@@ -246,7 +246,9 @@ def __init__(self, filepath, lidar_name=None, lidar2_name=None, pose2d_name=None
         if keyframes_name is not None:
             self.keyframes_id = names.index(keyframes_name) + 1
         if title_name is not None:
-            self.title_id = names.index(title_name) + 1
+            title_list = title_name.split(",")
```

Comment:
I was going to argue, that `argparse` can handle multiple arguments, but this modification seems to be reasonable, that I would accept even comma separated names of streams ...